rpm/SPECS/docker-ce.spec

```diff
@@ -84,6 +84,11 @@ export DOCKER_GITCOMMIT=%{_gitcommit_engine}
 mkdir -p /go/src/github.com/docker
 ln -s ${RPM_BUILD_DIR}/src/engine /go/src/github.com/docker/docker
 
+%if 1%{?_with_btrfs}
+DOCKER_BUILDTAGS+=' exclude_graphdriver_btrfs'
+export DOCKER_BUILDTAGS
+%endif
```

Comment:
Wouldn't this already be set though? https://github.com/docker/docker-ce-packaging/blob/27f81645524d59f5d3cce2b433125c19589c5b17/rpm/centos-8/Dockerfile#L14

Also, shouldn't this be using the `_without_btrfs` option? https://github.com/docker/docker-ce-packaging/blob/4cd3a7eca2f7887b15e98c2e67be1880f337d291/rpm/SPECS/docker-ce.spec#L3